dali/ft/filecopy.cpp

```diff
@@ -2873,6 +2914,16 @@ void FileSprayer::updateTargetProperties()
                 // TODO: Create DistributedFilePropertyLock for parts
                 curPart->lockProperties();
                 IPropertyTree& curProps = curPart->queryAttributes();
+
+                FilePartInfo & curHeaderSource = sources.item(whichHeaderInput);
+                curProps.setPropInt(FAheaderLength, curHeaderSource.xmlHeaderLength);
```

Comment:
Are these always getting set?  Would it be better to set them at the file level if they are all the same, then set them at the part level only if necessary?